system_core/datadog_checks/system_core/system_core.py

```diff
@@ -5,21 +5,36 @@
 from six import iteritems
 
 from datadog_checks.base import AgentCheck
+from datadog_checks.base.utils.platform import Platform
 
 
 class SystemCore(AgentCheck):
     def check(self, instance):
         instance_tags = instance.get('tags', [])
 
-        cpu_times = psutil.cpu_times(percpu=True)
-        self.gauge('system.core.count', len(cpu_times), tags=instance_tags)
+        # https://psutil.readthedocs.io/en/latest/#psutil.cpu_count
+        n_cpus = psutil.cpu_count()
+        self.gauge('system.core.count', n_cpus, tags=instance_tags)
```

Comment:
It looks like this is changing how `system.core.count` is collected. Should this be in a separate PR?

Reply:
I can separate them.